src/app/clusters/channel-server/channel-delegate.h

```diff
@@ -39,10 +39,10 @@ class Delegate
     virtual CHIP_ERROR HandleGetLineup(app::AttributeValueEncoder & aEncoder)         = 0;
     virtual CHIP_ERROR HandleGetCurrentChannel(app::AttributeValueEncoder & aEncoder) = 0;
 
-    virtual void HandleChangeChannel(const chip::CharSpan & match,
-                                     CommandResponseHelper<Commands::ChangeChannelResponse::Type> & responser) = 0;
-    virtual bool HandleChangeChannelByNumber(const uint16_t & majorNumber, const uint16_t & minorNumber)       = 0;
-    virtual bool HandleSkipChannel(const uint16_t & count)                                                     = 0;
+    virtual void HandleChangeChannel(CommandResponseHelper<Commands::ChangeChannelResponse::Type> & helper,
```

Comment:
Could we have a smaller code delta by not switching parameter order?

Reply:
this re-ordering was intentional in order to follow consistent conventions with this parameter